<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaVisualizzaSpettacoloAmministratore</class>
 <widget class="QWidget" name="VistaVisualizzaSpettacoloAmministratore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spettacolo - CineMax</string>
  </property>
  <widget class="QLabel" name="Sfondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 1,
        x2: 0, y2: 0,
        stop: 0 #210A00,
        stop: 1 #7D2100
    );
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelIconaFoto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Viste/Immagini/IconaFotoAmministratore.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitolo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Titolo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitoloPrincipale">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #D7320C;
}</string>
   </property>
   <property name="text">
    <string>Spettacolo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="ClickableLabel" name="labelHomeButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>63</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Viste/Immagini/HomeButtonAmministratore.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelGenere">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Genere:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitoloSpettacolo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGenereSpettacolo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrarioInizio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Orario Inizio:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrarioInizioSpettacolo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSala">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Sala:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSalaSpettacolo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelData">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDataSpettacolo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickableLabel" name="labelIndietroButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>63</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Viste/Immagini/IndietroButtonAmministratore.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrarioFine">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Orario Fine:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrarioFineSpettacolo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDurata">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Durata:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDurataSpettacolo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonModifica">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #961E00;
    color: #FF3C00;
    border: 2px solid #501400;
    border-radius: 6px;
}

QPushButton:focus {
    outline: none;
}

QPushButton:hover {
    background-color: #C83200;
}</string>
   </property>
   <property name="text">
    <string>Modifica</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStato">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Stato:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatoSpettacolo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrezzo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Prezzo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrezzoPunti">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Prezzo in punti:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrezzoSpettacolo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrezzoPuntiSpettacolo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonElimina">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #961E00;
    color: #FF3C00;
    border: 2px solid #501400;
    border-radius: 6px;
}

QPushButton:focus {
    outline: none;
}

QPushButton:hover {
    background-color: #C83200;
}</string>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
